packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# What's new with PatternFly 5.0 
+
+The following sections highlight the new features and updates included in this release. You can [view the full details for this release on GitHub](https://github.com/patternfly/patternfly-react/releases) and [refer to our upgrade guide](/get-started/upgrade) to update from PatternFly 4 to PatternFly 5.
+
+## Dark theme support
+
+While PatternFly has supported a dark theme in beta for some time, our dark theme has not been fully documented or tested. As part of the release of PatternFly 5, we were dedicated to thoroughly testing our beta implementation to ensure that it was ready for everyone. With this major release, we're excited to say that PatternFly's dark theme is fully vetted and supported. You can configure your applications' dark theme to update based on users’ system preferences.
+
+We are also currently working on introducing more robust theming capabilities in a future PatternFly major release. Going forward, all future themes will include a light and dark theme.
+
+There are a couple of ways to enable dark theme in your products. You can: 
+
+1. Add `patternfly-theme-dark.css` to a page and add the `.pf-theme-dark` class to the page's `<html>` element, or 
+2. Apply the `.pf-theme-dark` class to the browser's `prefers-color-scheme` media query. 
+
+For information regarding the shipping and implementation processes for dark theme, as well as general advice for its use read our [dark theme handbook.](https://docs.google.com/document/d/1mRYEfUoOjTsSt7hiqjbeplqhfo3_rVDO0QqMj2p67pw/edit?usp=sharing)
+
+## React 18 support
+
+We are committed to supporting the 2 most recent versions of React for PatternFly to keep up with the environments that our users work in. To this end, we are pleased to announce that PatternFly 5 is compatible with both React 17 and React 18.
+
+## Versioned CSS variables and class names
+
+We wanted to ensure that our variable naming conventions were clear and consistent going forward with our future release cycles. To keep things organized and clear, we updated our CSS variables and class names so that they are explicitly associated with the version of PatternFly that they align to. For PatternFly 5, we added a prefix of  `v5` to a number of our variable and class names.
+
+For more details, refer to [our upgrade guide](/get-started/upgrade#review-and-update-variable-and-class-names) and our [updated CSS classes and variables](developer-resources/global-css-variables) documentation.
+
+### Event handler parameter 
```

Comment:
I think you should remove one of the `#` in this title since it's not a sub heading of the one above it. Same for the Icons title